crypto/evp/names.c

```diff
@@ -90,6 +90,8 @@ void evp_cleanup_int(void)
 
     EVP_PBE_cleanup();
     OBJ_sigid_free();
+
+    evp_app_cleanup_int();
```

Comment:
Ah, now I know why I was confused.  I expected this call to appear in `OPENSSL_cleanup`, not here.  Frankly, I'm not sure this is the right spot...

Reply:
All EVP related cleanups should be grouped together and be called once in OPENSSL_cleanup, shouldn't they?

Reply:
Mm, yeah ok...